src/ripple/basics/Slice.h

```diff
@@ -22,11 +22,13 @@
 
 #include <ripple/basics/contract.h>
 #include <ripple/basics/strHex.h>
+#include <ripple/basics/Blob.h>
 #include <algorithm>
 #include <array>
 #include <cassert>
 #include <cstdint>
 #include <cstring>
+#include <iterator>
```

Comment:
Shouldn't need this. And when we are working with linear memory buffers we should not use iterators and the `<algorithm>` paradigm. We should just use `memcpy`.